### PR TITLE
Remove CHECK and FATAL in graphd

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: src/graph/YieldExecutor.cpp [hunk 1]
```diff
@@ -446,8 +446,9 @@ void YieldExecutor::feedResult(std::unique_ptr<InterimResult> result) {
 }
 
 void YieldExecutor::setupResponse(cpp2::ExecutionResponse &resp) {
-    CHECK(resp_ != nullptr);
-    resp = std::move(*resp_);
+    if (resp_ != nullptr) {
```
Comment (monadbobo) [inline src/graph/YieldExecutor.cpp]: Here maybe an empty resp should be set to return to the caller, similar to the GoExecutor :: setupResponse method？

Comment (laura-ding) [reply src/graph/YieldExecutor.cpp]: `GoExecutor::setupResponse ` create an empty object is redundant, wdyt?

File: src/graph/AggregateFunction.h [hunk 2]
```diff
@@ -71,7 +71,8 @@ class GroupByHash {
                 return 0;
             }
             default: {
-                LOG(FATAL) << "Untreated value type: " << static_cast<int32_t>(col.getType());
+                LOG(ERROR) << "Untreated value type: " << static_cast<int32_t>(col.getType());
+                return 0;
```
Comment (whitewum) [inline src/graph/AggregateFunction.h]: return 0 or other error code?

**Comment by jude-zhu** [comment]
close #1495

**Comment by jude-zhu** [comment]
#1184

**Pull request merged**